README.md

```diff
@@ -1,2 +1,73 @@
-# Template-Webpack-Linters
- Template for new repositories with webpack and linters
+# Pokedex
+Interactive Pokedex, with all the information for the 300 pokemons, it's abilities and attritubes, and more! Get your Pokedex and become a master Pokemon!
+
+## Built With
+
+GitHub,
+GitFlow,
+Visual Studio Code,
+HTML,
+CSS,
+Javascript,
+Webpack,
+Jest,
+Gitflow
+
+## Getting Started
+
+Welcome to our new Pokedex! Working with new methodology por interactive flow of data, linked with an API to collect information on all Pokemons. Saving the data locally and rendering all cards and each of it's details. You can like your favorites and also leave comments! Come check it out!
+To get a local copy up and running please download the zip file or clone it via git
+
+## Live Demo
+
+[Click here!](https://sntgopl.github.io/API-Pokemon-Instagram/dist/)
+
+### Live Presentation
+
+[video](https://drive.google.com/file/d/1xn30tRNJqa6JnuzH7G93YGM7DtiINMnF/view?usp=sharing)
+
+### Clone and review it
+
+Run the following commands to clone and run it.
+
+### To clone the repository
+
+https://github.com/sntgopl/API-Pokemon-Instagram.git
+
+
+### To Install Packages
+
+npm install
+
+### To run project on local server
+
+npm start
+
+## Author
+
+Santiago's:
+- GitHub: [@sntgopl](https://github.com/sntgopl)
+- Linkedin: [Santiago Pons Lezica](https://www.linkedin.com/in/santiago-pons-lezica-923747241/)
+
+Camilo's:
+- GitHub: [@camilovelag](https://github.com/camilovelag)
+- Twitter: [@camilovelag](https://twitter.com/camilovelag)
+- LinkedIn: [camilovelag](https://linkedin.com/in/camilovelag)
+
+## Contributing
+
+Contributions, issues, and reviews are very welcome! 
+
+## Support
+
+Give a ⭐ if you like this project!
+
+## Acknowledgements
+
+Microverse Community
+GitHub Repositories
+Minimalist Leaderboard list
+
+## 📝 License
+
+This project is MIT licensed.
```

Comment:
- [ ] [OPTIONAL] It would be great if you add a link to your license file [like this](URL)